### PR TITLE
Add auto sort

### LINKED ISSUE #121: Add function to auto-sort claims within insurance plan
Currently, the claims are sorted in the order in which they are added. They should instead appear in lexicographical order which would be more appropriate. They should be sorted automatically once the new claim is added.

### PR DESCRIPTION
**Pull request by RezwanAhmed123**
Currently, the claims are sorted in the order in which they are added. They should instead appear in lexicographical order which would be more appropriate. They should be sorted automatically once the new claim is added.

Fixed some weird bugs with adding of more than 1 claim, as well as some minor modifications to the close claim command.

Fixed #121

### CODE REVIEW

File: src/main/java/seedu/address/model/client/insurance/InsurancePlan.java [hunk 1]
```diff
@@ -50,13 +51,32 @@ public Claim getClaim(String claimId) throws ClaimException {
...
      *
      * @param claim The claim to be removed.
      */
     public void removeClaim(Claim claim) {
         claims.remove(claim);
+        sortClaims();
+    }
+
+    /**
+     * Sorts the claims of this insurance plan only based on claim status then by claim id.
```
Comment (zi-yii) [inline src/main/java/seedu/address/model/client/insurance/InsurancePlan.java:75]: Maybe using "client ID" would be better to follow how we write insurance id as "insurance ID" in javadoc

File: src/main/java/seedu/address/model/client/insurance/InsurancePlansManager.java [hunk 2]
```diff
@@ -75,7 +75,7 @@ public ArrayList<InsurancePlan> getInsurancePlans() {
      *                                the client's list of insurance plans.
      */
     public InsurancePlan getInsurancePlan(int insuranceId) throws InsurancePlanException {
-        for (InsurancePlan plan : insurancePlans) {
+        for (InsurancePlan plan : this.insurancePlans) {
```
Comment (zi-yii) [inline src/main/java/seedu/address/model/client/insurance/InsurancePlansManager.java:78]: Did this cause any bugs? Would we need to make the same change for the other methods in InsurancePlansManager as well?

Comment (RezwanAhmed123) [reply src/main/java/seedu/address/model/client/insurance/InsurancePlansManager.java:78]: no i was just being specific only, this shouldnt be an issue normally unless theres another object named the same within that local scope.

File: src/main/java/seedu/address/model/client/insurance/InsurancePlan.java (hunk 1 above)
Comment (RezwanAhmed123) [reply src/main/java/seedu/address/model/client/insurance/InsurancePlan.java:75]: that makes sense, good catch!

File: src/main/java/seedu/address/model/client/insurance/InsurancePlansManager.java (hunk 2 above)
Comment (zi-yii) [reply src/main/java/seedu/address/model/client/insurance/InsurancePlansManager.java:78]: Rights, okie noted!

**Review by zi-yii — APPROVED** [review]
LGTM, merging as tests will be implemented in a later PR

**Pull request merged**